### ISSUE TITLE
Design cleanup

### ISSUE THREAD

**Issue by the-swerve** [openoakland/soft-story-buildings#12]
Do a styling cleanup of the layout

**Comment by the-swerve** [comment]
Did a set of basic styling to make it a little easier on the eyes. Still doesn't look great on mobile sizes, but it works

**Comment by the-swerve** [comment]
Sorry, I hadn't noticed that @criscristina had done some stylings in a couple branches. Feel free to roll this back

**Comment by criscristina** [comment]
No worries.  You can just merge my style branch.

**Comment by the-swerve** [comment]
I'll merge in your styles asap

**Comment by the-swerve** [comment]
It's merged into master. I archived the styles I did independently in stylesheets/alt.css in case we want to include them back. I think it still needs a good amount of style work. I can do branch & pull style changes from here on out; my instinct with small projects is to just start pushing stuff directly.

https://github.com/openoakland/soft-story-buildings/commit/66eb36cfbaac54e474ed39ddb6531e74aa23de02
